kitsune/questions/static/questions/js/components/UserAuth.jsx

```diff
@@ -1,97 +1,49 @@
 /* globals React:false */
 import AAQStep from './AAQStep.jsx';
 import UserAuthActions from '../../../users/js/actions/UserAuthActions.es6.js';
+import {authStates} from '../../../users/js/constants/UserAuthConstants.es6.js';
+import LoginForm from '../../../users/js/components/LoginForm.jsx';
+import RegisterForm from '../../../users/js/components/RegisterForm.jsx';
 
 export default class UserAuth extends AAQStep {
   constructor(props) {
     super(props);
-    this.state = {
-      signingIn: false,
-    };
```

Comment:
The state held by `signingIn` was moved to the state machine in `UserAuthStore.`